<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактирование базы</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/viewmag_4571.png</normaloff>icons/viewmag_4571.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Список вирусов, определенных в базе</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QListWidget" name="listVirus"/>
            </item>
            <item>
             <widget class="QWidget" name="widget_9" native="true">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>28</height>
               </size>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <property name="margin">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="btnDeleteSelect">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>28</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>28</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Удалить</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>icons/cancel_7207.png</normaloff>icons/cancel_7207.png</iconset>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnRename">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>28</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>28</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Переименовать</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>icons/filesaveas_8370.png</normaloff>icons/filesaveas_8370.png</iconset>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QTextEdit" name="tbVirusName"/>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_11" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_11">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_4" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="widget_7" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <property name="margin">
                <number>0</number>
               </property>
               <item>
                <widget class="QGroupBox" name="groupBox_3">
                 <property name="title">
                  <string>Патерны расширений</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_9">
                  <property name="margin">
                   <number>9</number>
                  </property>
                  <item>
                   <widget class="QTextEdit" name="tbExiption">
                    <property name="verticalScrollBarPolicy">
                     <enum>Qt::ScrollBarAlwaysOn</enum>
                    </property>
                    <property name="horizontalScrollBarPolicy">
                     <enum>Qt::ScrollBarAlwaysOn</enum>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_6" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <property name="margin">
                <number>0</number>
               </property>
               <item>
                <widget class="QGroupBox" name="groupBox_2">
                 <property name="title">
                  <string>Патерны названий</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_8">
                  <property name="margin">
                   <number>9</number>
                  </property>
                  <item>
                   <widget class="QTextEdit" name="tbName">
                    <property name="verticalScrollBarPolicy">
                     <enum>Qt::ScrollBarAlwaysOn</enum>
                    </property>
                    <property name="horizontalScrollBarPolicy">
                     <enum>Qt::ScrollBarAlwaysOn</enum>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_5" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QGroupBox" name="groupBox_4">
              <property name="title">
               <string>Патерны содержания</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_10">
               <item>
                <widget class="QTextEdit" name="tbFile">
                 <property name="verticalScrollBarPolicy">
                  <enum>Qt::ScrollBarAlwaysOn</enum>
                 </property>
                 <property name="horizontalScrollBarPolicy">
                  <enum>Qt::ScrollBarAlwaysOn</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>46</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>45</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_10" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox_5">
           <property name="title">
            <string/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="leftMargin">
             <number>9</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>9</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="btnAddNewVirus">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>28</height>
               </size>
              </property>
              <property name="text">
               <string>Добавить объект</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>icons/binary_1650.png</normaloff>icons/binary_1650.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="tbNewVirus">
              <property name="minimumSize">
               <size>
                <width>255</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>28</height>
               </size>
              </property>
              <property name="text">
               <string>Название вируса</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_12" native="true">
        <property name="minimumSize">
         <size>
          <width>129</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_8" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="btnApply">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>38</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>38</height>
            </size>
           </property>
           <property name="text">
            <string>Применить</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>icons/ok_5764.png</normaloff>icons/ok_5764.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnClose">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>38</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>38</height>
            </size>
           </property>
           <property name="text">
            <string>Выход</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>704</x>
     <y>481</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAddNewVirus</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>addNewVirus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>481</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDeleteSelect</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>deleteCurrentRow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>428</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listVirus</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>Form</receiver>
   <slot>itemChacked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRename</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>renameVirus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnApply</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>apply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>598</x>
     <y>481</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addNewVirus()</slot>
  <slot>deleteCurrentRow()</slot>
  <slot>itemChacked()</slot>
  <slot>renameVirus()</slot>
  <slot>apply()</slot>
 </slots>
</ui>
